docs/api/index.rst

```diff
@@ -16,3 +16,9 @@ API                     Specification version(s)
 Product API             `0.0.2 <https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/maykinmedia/open-producten/v0.0.2/src/producten-openapi.yaml>`__
 Producttype API         `0.0.2 <https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/maykinmedia/open-producten/v0.0.2/src/producttypen-openapi.yaml>`__
 ======================  ==========================================
+
+In addition, Open Producten requires access to a `Notificaties API`_. Open Producten uses
+`Open Notificaties`_ by default.
```

Comment:
Notifications can be disabled via the `NOTIFICATIONS_DISABLED` envvar, so I don't think it's a necessarily a requirement. It might be good to mention that it can be disabled (and add a link to the envvar docs)